src/main/java/slimeknights/tconstruct/tools/data/ModifierRecipeProvider.java

```diff
@@ -352,7 +384,34 @@ private void addModifierRecipes(Consumer<IFinishedRecipe> consumer) {
                          .setMaxLevel(2)
                          .setAbilitySlots(1)
                          .build(consumer, prefixR(TinkerModifiers.reach, upgradeFolder));
-
+    // block transformers
+    ModifierRecipeBuilder.modifier(TinkerModifiers.shovelTransform.get())
+                         .setTools(new IngredientWithout(Ingredient.fromTag(TinkerTags.Items.HARVEST), Ingredient.fromItems(TinkerTools.mattock, TinkerTools.excavator)))
+                         .addInput(TinkerModifiers.ancientShovelHead.get())
+                         .setMaxLevel(1)
+                         .setUpgradeSlots(1)
```

Comment:
These transform modifiers let you essentially replace the need for another tool (notably a kama, as thats the mosst useful one), so I am inclined to call them abilities.